model/websocket_request.go

```diff
@@ -28,6 +28,19 @@ func (o *WebSocketRequest) ToJson() string {
 	return string(b)
 }
 
+func (o *WebSocketRequest) Clone() *WebSocketRequest {
+	buf, err := json.Marshal(o)
+	if err != nil {
+		panic(err)
```

Comment:
Using panics is not a good practice. Let's return the error up to the caller, and `mlog.Warn` it and skip sending it to the channel in that case.

Reply:
Agree, copied the pattern from another model, but see no reason to just improve

Reply:
Fixed in https://github.com/mattermost/mattermost-server/pull/18151/commits/41cc7c22a2740711993d4ecba01cc6a5c0245580